src/main/java/src/main/java/EventModel.java

```diff
@@ -0,0 +1,36 @@
+package src.main.java;
+
+public class EventModel {
+    private String nameEvent;
+    private String body;
```

Comment:
```suggestion
    private JrsObject body;
```
Comme on sait que le body sera toujours un objet JSON, c'est bien de mettre cette information dans le typage, c'est pour ça que je te suggère la modification ci-dessus. String c'est trop "flou" comme typage.

Peut être que `JrsObject` n'est pas le bon type, vérifie quel est le type pour un objet json dans la librairie Jackson.